EDU-3638 feat: add object storage reference documentation (#569)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/build/edge-application/object-storage.mdx

---
title: Object Storage
description: Azion Object Storage is a scalable and secure storage service designed to integrate object storage with the Azion Edge Computing Platform using the S3 standard.
meta_tags: object storage, storage, cloud, s3, bucket, file storage
namespace: documentation_products_edge_application_object_storage

> permalink: /documentation/products/build/edge-application/object-storage/

Author: ```suggestion
permalink: /documentation/products/store/object-storage/
```

---

Azion **Object Storage** is a scalable and secure storage service designed to integrate object storage with the Azion Edge Computing Platform using the S3 standard. Buckets can be used as origins for edge applications or as directories for real-time object upload.

Alongside bucket creation, you possess complete control over storage allocation. This includes management of read/write permissions and the ability to upload, change, and delete objects.

:::note
This feature is available via the [Azion API](https://api.azion.com), [Edge Runtime](/en/documentation/devtools/runtime/overview/), and [Azion CLI](/en/documentation/products/azion-cli/overview/).
:::

## Implementation

| Scope | Resource |
| --- | --- |

> | Manage a bucket | [How to create and manage an object storage bucket](/en/documentation/products/guides/how-to-create-and-manage-bucket/) |

Author: ```suggestion
| Manage a bucket | [How to create and modify an Object Storage bucket](/en/documentation/products/guides/how-to-create-and-modify-bucket/) |
| Upload and download objects | [How to upload and download objects from an Azion Object Storage bucket](/en/documentation/products/guides/how-to-upload-and-download-objects-from-bucket/) |
| Runtime API | [Object Storage API](/en/documentation/devtools/runtime/api-reference/storage/) |
```

> | Gerenciar um bucket | [How to create and manage an object storage bucket](/en/documentation/products/guides/how-to-create-and-manage-bucket/) |

Author: ```suggestion
| Manage a bucket | [How to create and manage an object storage bucket](/en/documentation/products/guides/how-to-create-and-manage-bucket/) |
```
